source/features/next-scheduled-github-action.tsx

```diff
@@ -65,14 +65,19 @@ async function init(): Promise<false | void> {
 
 		const nextTime = parseCron.nextDate(workflows[workflowName]);
 		if (nextTime) {
-			workflowListItem.append(<em>(next <relative-time datetime={nextTime.toString()}/>)</em>);
+			const relativeTime = <relative-time datetime={nextTime.toString()}/>;
+			workflowListItem.append(<em>(next {relativeTime})</em>);
+			workflowListItem.parentElement!.classList.add('tooltipped', 'tooltipped-n');
```

Comment:
Let's show it on the right, like a continuation of the line itself
```suggestion
			workflowListItem.parentElement!.classList.add('tooltipped', 'tooltipped-e');
```

<img width="336" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/109883950-cda82780-7c41-11eb-95a6-ea06773625c1.png">